packages/store-ui/src/atoms/Input/Input.tsx

```diff
@@ -0,0 +1,32 @@
+import type { InputHTMLAttributes } from 'react'
+import React, { forwardRef } from 'react'
+
+export interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
+  isError?: boolean
```

Comment:
Why not just `error` to be close to `disabled`, for example? (Or call `disabled` as `isDisabled`, which I'm not a big fan of)

Reply:
I'm still not sure about that decision.  but my reasoning was: disabled is a html selector, isError is a component state at the library level.  So what is html continues the same names, and states start with is.  What do you think?

Reply:
I agree with @matheusps on that one. I prefer having a single API